single-sign-on/index.md

```diff
@@ -169,9 +169,7 @@ Admins can force users to authenticate with Docker Desktop by provisioning a reg
 
 ## Manage users when SSO is enabled
 
-To add a member to your organization:
-1. Create an account for your members in your IdP.
-2. Add and invite your members to your organization.
+You don’t need to manually add users to your organization in Docker Hub. You just need to make sure an account for your users exists in your IdP and then invite them to your organization using the Invite Member option in Docker Hub.
```

Comment:
Suggest we delete referring to inviting members through the Invite member tab when SSO is enabled

Reply:
Thanks Usha! Updated here and in FAQs with new message